src/components/TablePoc/userTableConfig.ts

```diff
@@ -0,0 +1,34 @@
+
+import {TableColumnConfig} from './TableColumnConfigTypes';
+import {UserDataTypes} from './UserData.types';
+
+export const userTableConfig: TableColumnConfig<UserDataTypes>[] = [
+    {
+        key: 'firstName',
+        header: 'Prénom'
+    },
+    {
+        key: 'lastName',
+        header: 'Nom',
+        cell: value => (value as string).toUpperCase()
```

Comment:
For the first iteration, we will not change the value of the data. We will just display the value.